### ISSUE TITLE
compose support?

### ISSUE THREAD

**Issue by mochadwi** [trevjonez/Kontrast#17]

**Comment by trevjonez** [comment]
good idea, though i've not been able to provide the time to the project which it deserves. have you given a look at the square alternative? https://github.com/cashapp/paparazzi

**Comment by mochadwi** [comment]
there's an open issue related to compose in their repo: https://github.com/cashapp/paparazzi/issues/225
